modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/chapters/ChaptersViewModel.kt

```diff
@@ -37,15 +41,22 @@ class ChaptersViewModel
 
     data class UiState(
         val chapters: List<ChapterState> = emptyList(),
+        val selectedIndexes: List<Int> = emptyList(),
```

Comment:
I think in the long run it will be simpler to have a property on a chapter like `Chapter.isSelected`. Then we can update chapters with `chapter.copy(isSelected = value)` without a need to track it in a separate index list. We can also avoid passing `checked` to `ChapterRow` this way.

Reply:
Good point, I updated it in https://github.com/Automattic/pocket-casts-android/pull/1827/commits/da2f7a287fbb9936e7e3b45eb0988e44f1647e65